File: API/KenJobs.Api/KenJobs.Api/Models/JobSearchModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace KenJobs.Api.Models
{
    public class JobSearchModel
    {
        public string Keyword { get; set; }
        public string Location { get; set; }
        public int? Experience { get; set; }
    }
}

File: UI/WEB/src/Models/JobSearchModel.ts
import PaginationModel from "./PaginationModel";
import { GridRequest } from "./GridModel";

export default class JobSearchModel {
    Keyword: string = "";
    Location: string = "";
    Experience?: number = 0;
    //paginationdata: PaginationModel=new PaginationModel();
    JobSearchRequest: GridRequest = new GridRequest();    
}